### ISSUE TITLE
Any attributes passed to the component is being ignored

### ISSUE THREAD

**Issue by karimshalapy** [jfgodoy/vite-plugin-solid-svg#43]
Whenever I try to pass `role`, `id`, or `class` no error shows in the IDE and the output HTML doesn't contain any of the inputted attributes.

I thought it might be SVGO, so I disabled it entirely, and the same issue exists.

Not sure if I am doing something wrong or if this is something that needs to be addressed.

The config:

```ts
solidSvg({
  defaultAsComponent: false,
  svgo: {
    svgoConfig: {
      removeViewBox: false,
    },
  },
}),
```

Package version:

```json
"vite-plugin-solid-svg": "^0.7.0",
"solid-js": "^1.8.5",
```

**Comment by jfgodoy** [comment]
hi @karimshalapy, I can't reproduce the problem.
![image](https://github.com/jfgodoy/vite-plugin-solid-svg/assets/731635/60fd3dd0-4638-4637-9541-6ef39657272e)
![image](https://github.com/jfgodoy/vite-plugin-solid-svg/assets/731635/370768fb-16f1-4e2e-b0a4-73d9c4babdf4)

If you can replicate it in a demo project, I can help you find the problem.

**Comment by karimshalapy** [comment]
@jfgodoy It's happening currently in our project: https://github.com/KL13NT/badeel
If this is too much and you don't want to check the issue there, I've made a fresh solid-js project and recreated the issue there.
You can find that in this repo: https://github.com/karimshalapy/solid-svg-issue
Or the codesandbox here: https://codesandbox.io/p/github/karimshalapy/solid-svg-issue/main?file=%2Fsrc%2FApp.tsx%3A13%2C1-14%2C1&workspaceId=cf960261-2a33-4e58-a54e-1d60ab28445c

I am not sure why it happens to be honest, I thought at first it's conflicting plugins, but in the fresh install it's still there.

**Comment by jfgodoy** [comment]
Hi @karimshalapy, the problem was related to this issue: https://github.com/jfgodoy/vite-plugin-solid-svg/pull/38. It was affecting SVGs where the `<svg` was in one line and the props were in another. The fix has been published in version 0.8.0.
